tools/delete_candidate.php

```diff
@@ -122,27 +122,42 @@
     break;
 }
 
-/*
+/**
  * Prints the usage and example help text and stop program
+ *
+ * @return void
  */
 function showHelp()
 {
-    echo "*** Delete Candidate Info ***\n\n";
-
-    echo "Usage: php delete_candidate.php delete_candidate CandID PSCID [confirm]\n";
-    echo "Example: php delete_candidate.php delete_candidate 965327 dcc0007\n";
-    echo "Example: php delete_candidate.php delete_candidate 965327 dcc0007 confirm\n";
-    echo "Example: php delete_candidate.php delete_candidate 965327 dcc0007 tosql\n\n";
-
-    echo "When the 'tosql' function is used, the SQL file exported will be located \n".
-        "under the following path: loris_root/project/tables_sql/DELETE_candidate_CandID.sql\n\n";
-
-    die();
+    echo <<<USAGE
+*** Delete Candidate Info ***
+
+Usage: php delete_candidate.php delete_candidate CandID PSCID [confirm]
+Example: php delete_candidate.php delete_candidate 965327 dcc0007
+Example: php delete_candidate.php delete_candidate 965327 dcc0007 confirm
+Example: php delete_candidate.php delete_candidate 965327 dcc0007 tosql
+
+When the 'tosql' function is used, the SQL file exported will be located
+under the following path: 
+    loris_root/project/tables_sql/DELETE_candidate_CandID.sql
+USAGE;
+    die;
 }
-/*
- * All tables with entries to be deleted here are only those with FOREIGN KEY relations to `candidate`.
- * All other tables with FOREIGN KEY relations to these tables (second-level relations) should
- * have actions on delete specified in the database schema
+/**
+ * All tables with entries to be deleted here are only those with FOREIGN KEY
+ * relations to `candidate`.
+ * All other tables with FOREIGN KEY relations to these tables
+ * (second-level relations) should have actions on delete specified in the
+ * database schema
```

Comment:
```suggestion
 * database schema.
```
I missed that one initially.